resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
                 "lost_cats": ["r_c"]
             }
         ]
+    },
+    {
+        "patrol_id": "gen_train_amentorswisdom",
+        "biome": ["Any"],
+        "season": ["Any"],
+        "types": ["training"],
+        "tags": ["platonic","trust","respect","dislike","comfort"],
+        "patrol_art": "train_general_intro",
+        "min_cats": 2,
+        "max_cats": 2,
+        "min_max_status": {
+            "normal adult": [0, 1],
+            "all apprentices": [0, 1]
+        },
+    
+        "weight": 20,
+        "chance_of_success": 50,
+        "intro_text": "p_l is surprised when app1 makes a mistake on a simple move p_l thought {PRONOUN/app1/subject} had mastered. Before p_l can say anything, app1 wails and runs out of the training grounds. When p_l finds {PRONOUN/app1/poss} huddled in camp, {PRONOUN/app1/subject} {VERB/app1/sob/sobs} out a confession: {PRONOUN/app1/subject} will never be ready to be a warrior.",
```

Comment:
You may want to avoid the word 'wail', as it makes the apprentice sound very young, not what I'd expect from a 12 moon old cat - tho very appropriate for a 6 moon old one. But the wording has to work for both